config/helm-manifest/kustomization.yaml

```diff
@@ -29,7 +29,7 @@ patchesJson6902:
       - op: replace
         path: /spec/template/spec/
         value: |
-          {{- if .Values.global.imagePullSecrets }}
+          {{- if and .Values.global .Values.global.imagePullSecrets }}
```

Comment:
is this valid? just asking for my learning

Reply:
yes it is needed. This is the nil check. Otherwise the helm chart fails if theres no global stanza

Reply:
also thats the right syntax